package.json

```diff
@@ -1,8 +1,52 @@
 {
   "name": "esptool-js",
   "version": "0.1.0",
+  "module": "lib/index.js",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "files": [
+    "lib",
+    "bundle.js"
+  ],
+  "scripts": {
+    "build": "npm run clean && tsc",
+    "clean": "rimraf lib bundle.js",
+    "format": "prettier --write \"src/**/*.ts\" \"src/**/*.js\"",
+    "lint": "eslint . --ext .ts",
+    "lintAndFix": "eslint . --ext .ts --fix",
+    "prepare": "npm run build",
+    "test": "echo \"Error: no test specified\"",
+    "prepublishOnly": "npm run test && npm run lint",
```

Comment:
FYI lint currently fails, but I suggest to run prettier in follow-up PR which also adds a CI step so every PR is checked.